util/execdetails.go

```diff
@@ -133,12 +158,14 @@ type CommitDetails struct {
 	LocalLatchTime         time.Duration
 	Mu                     struct {
 		sync.Mutex
-		CommitBackoffTime   int64
-		BackoffTypes        []string
-		SlowestReqTotalTime time.Duration
-		SlowestRegion       uint64
-		SlowestStoreAddr    string
-		SlowestExecDetails  TiKVExecDetails
+		// The total backoff time used in both the prewrite and commit phases.
+		CommitBackoffTime    int64
+		PrewriteBackoffTypes []string
+		CommitBackoffTypes   []string
+		// The prewrite requests are executed concurrently so the slowest request information would be recorded.
+		SlowestPrewrite ReqDetailInfo
+		// It's recorded only when the commit mode is 2pc.
+		CommitPrimary ReqDetailInfo
```

Comment:
Shall we also observe the number of prewrite/commit requests as weel?

Reply:
A new filed `PrewriteReqNum` is added.